mastering-plone/thirdparty_behaviors.rst

```diff
@@ -39,9 +36,7 @@ The training buildout has a section ``[sources]`` that tells buildout to downloa
     collective.behavior.banner = git https://github.com/collective/collective.behavior.banner.git pushurl=git@github.com:collective/collective.behavior.banner.git rev=7c13285
 
 Pinning the revision saves us from being surprised by changes in the code we might not want.
-
 You can also pin a branch or a tag.
-
```

Comment:
This blank line should be restored to separate these blocks into two paragraphs.

Reply:
Done.